lib/galaxy/webapps/galaxy/api/user_preferences.py

```diff
@@ -783,14 +798,7 @@ def get_filter_mapping(db_filters, config_filters, factory):
             }
 
     @expose_api
-    def toolbox_filters(self, trans, cntrller='user_preferences', **kwd):
-        """
-        API call for fetching toolbox filters data
-        """
-        return self.tool_filters(trans, **kwd)
-
-    @expose_api
-    def edit_toolbox_filters(self, trans, cntrller='user_preferences', **kwd):
+    def __edit_toolbox_filters(self, trans, cntrller='user_preferences', **kwd):
```

Comment:
I don't think something with an ``@expose_api`` tag should be private like this. If it is unused now I'd just remove it.